ddtrace/contrib/kafka/patch.py

```diff
@@ -27,6 +28,17 @@
 _DeserializingConsumer = (
     confluent_kafka.DeserializingConsumer if hasattr(confluent_kafka, "DeserializingConsumer") else None
 )
+_SerializationContext = (
+    confluent_kafka.serialization.SerializationContext
+    if hasattr(confluent_kafka.serialization, "SerializationContext")
```

Comment:
What's the minimum version where `confluent_kafka.serialization` has the class `SerializationContext`? 

nit: Using the version of the integration here could be a little easier to maintain than a hasattr check.

Reply:
v1.4.0 is the version, I'll make this change!